pkg/flow/internal/controller/loader.go

```diff
@@ -87,7 +88,8 @@ func (l *Loader) Apply(parentScope *vm.Scope, blocks []*ast.BlockStmt) diag.Diag
 		diags = append(diags, multierrToDiags(err)...)
 		return diags
 	}
-
+	// Copy the original graph
```

Comment:
Can you add detail in the comment to explain why we're doing this? (so we can expose the non-reduced graph to users to examine original connections)